src/core/time.d

```diff
@@ -4329,6 +4361,48 @@ string numToString(long value) @safe pure nothrow
         assert(0, "Something threw when nothing can throw.");
 }
 
+unittest
+{
+    string a = numToString(1);
+    assert(a == "1");
+
+    a = numToString(-1);
+    assert(a == "-1");
+}
+
+/++ A six digit double to string converstion function
++/
+string numToString(double value) @safe pure nothrow
+{
+    immutable negative = value < 0;
+    string result = (negative ? "-" : "") ~ numToString(cast(long)value);
+    result ~= '.';
+    result ~= numToString(cast(long)(_abs((value - cast(long)value) * 1000000)));
```

Comment:
This is only used in unit tests. I'm thinking this needs to go into a `version(unittest)` block. core.time shouldn't be in the business of providing string conversion functions (and take off the ddoc for it)

`numToString(long)` is a necessary evil (not sure that should be publicly documented either, maybe it actually should go into core.internal) because of the requirements for purity during time printing, but this one isn't used in the actual implementation.